src/main.go

```diff
@@ -221,3 +227,24 @@ func main() {
 	log.Fatal(http.ListenAndServe(port, nil))
 
 }
+
+func setupLogger() {
+	currTime := time.Now()
+	currFileName := fmt.Sprint(logFilePrefix, currTime.Format("2006-01-02#15:04:05"), ".log")
+	user, err := user.Current()
+	if err != nil {
+		fmt.Printf("cannot access current user data\n")
+		return
+	}
+	homePath := user.HomeDir
+	logFilePath := homePath + "/" + currFileName
```

Comment:
Please store the logs inside the `storage/logs/` _(create one by code if not found)_ directory in bench-routes. We do not want to overcrowd the `/home`.

Reply:
@Harkishen-Singh In future if bench-routes is shipped as a standalone binary (without source) then the path has to be changed. hence I chose current user's home for dumping logs. what do you think ?

Reply:
@Pratik32 we are yet to plan in that part. For now, we can make a new folder in the `home` as **br-logs** and keep all the logs in it in. That would be better.

Reply:
Okay. will modify code accordingly.